ibm-cloud-pak-for-data/azuredeploy.json

```diff
@@ -1022,6 +1025,63 @@
 				"OpenShiftDeployment",
 				"[variables('deployOpenshiftExt')]"
 			],
+			"properties": {
+				"mode": "Incremental",
+				"templateLink": {
+					"uri": "[variables('preloadImagesDeploymentTemplateUrl')]",
+					"contentVersion": "1.0.0.0"
+				},
+				"parameters": {
+					"assembly": {
+						"value": "lite"
+					},
+					"uploadImagesScriptUrl": {
+						"value": "[variables('uploadImagesScriptUrl')]"
+					},
+					"uploadImagesScriptFileName": {
+						"value": "[variables('uploadImagesScriptFileName')]"
+					},
+					"redHatTags": {
+						"value": "[variables('redHatTags')]"
+					},
+					"adminUsername":{
+						"value": "[parameters('adminUsername')]"
+					},
+					"ocuser": {
+						"value": "[parameters('openshiftUsername')]"
+					},
+					"ocpassword": {
+						"value": "[parameters('openshiftPassword')]"
+					},
+					"bastionHostname": {
+						"value": "[variables('bastionHostname')]"
+					},
+					"projectName": {
+						"value": "[parameters('projectName')]"
+					},
+					"location": {
+						"value": "[parameters('location')]"
+					},
+					"clusterName": {
+						"value": "[parameters('clusterName')]"
+					},
+					"domainName": {
+						"value": "[parameters('dnsZone')]"
+					}
+				}
+			}
+		},
+		{
+			"name": "CloudPakLiteDeployment",
```

Comment:
See: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#sort-order-of-properties for the proper sort order of properties